src/app/child-dev-project/previous-schools/previous-schools.component.ts

```diff
@@ -40,9 +63,11 @@ export class PreviousSchoolsComponent
     }
     if (panelConfig.config?.columns) {
       this.columns = panelConfig.config.columns;
+      this.columns.push(this.isActiveIndicator);
     }
     this.child = panelConfig.entity as Child;
     this.loadData(this.child.getId());
+    this.subscribeEntityUpdates(ChildSchoolRelation);
```

Comment:
```suggestion
    this.subscribeToChildSchoolRelationUpdates();
```
In this case it is not really necessary to pass this as an argument. This makes a little more clear what is happening.